app/views/products/edit.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1> Edit a vendor: </h1>
+
+
+<% if @product.errors.any? %>
+  <ul>
+    <% @product.errors.each do |column, message| %>
+      <li>
+        <%= column.capitalize %> <%= message %>
+      </li>
+    </ul>
+    <% end %>
+<% end %>
+
```

Comment:
Could consider pulling out all of the error-related html that is being repeated into a partial.